src/overlays/ovl_i2/fox_sx.c

```diff
@@ -1,81 +1,1986 @@
+                player->camAt.y = player->pos.y * player->unk_148 + 20.0f;
+                player->camAt.z = player->unk_138;
+                player->camAt.x = player->camEye.x = player->pos.x;
+                D_80178488 = 1;
+                for (i = 0; i < 3; i++) {
+                    Object_Kill(&gActors[i].obj, &gActors[i].sfxPos);
+                }
+            }
+            break;
+    }
+    Math_SmoothStepToF(&player->camEye.x, D_80177978, D_80177A48[0], 20000.0f, 0);
+    Math_SmoothStepToF(&player->camEye.y, D_80177980, D_80177A48[0], 20000.0f, 0);
+    Math_SmoothStepToF(&player->camEye.z, D_80177988, D_80177A48[0], 20000.0f, 0);
+    Math_SmoothStepToF(&player->camAt.x, D_801779A0, D_80177A48[0], 20000.0f, 0);
+    Math_SmoothStepToF(&player->camAt.y, D_801779B8, D_80177A48[0], 20000.0f, 0);
+    Math_SmoothStepToF(&player->camAt.z, D_801779C0, D_80177A48[0], 20000.0f, 0);
+    Matrix_RotateY(gCalcMatrix, (player->unk_0E8 + 180.0f) * M_DTOR, 0U);
+    Matrix_RotateX(gCalcMatrix, -(player->unk_0E4 * M_DTOR), 1U);
+    src.x = 0.0f;
+    src.y = 0.0f;
+    src.z = player->unk_0D0;
+    Matrix_MultVec3fNoTranslate(gCalcMatrix, &src, &dest);
+    player->vel.x = dest.x;
+    player->vel.z = dest.z;
+    player->vel.y = dest.y;
+    player->pos.x += player->vel.x;
+    player->pos.y += player->vel.y;
+    player->pos.z += player->vel.z;
+    player->unk_138 = player->pos.z + player->unk_08C;
+}
+
+void func_i2_801944D4(Actor* actor, s32 arg1) {
+    Vec3f srcA;
+    Vec3f destA;
+    Vec3f srcB;
+    Vec3f destB;
+    Player* player = &gPlayer[0];
+
+    Matrix_RotateY(gCalcMatrix, player->unk_0E8 * M_DTOR, 0U);
+    srcA.x = D_i2_80195710[arg1];
+    srcA.y = D_i2_80195720[arg1];
+    srcA.z = D_i2_80195730[arg1];
+    srcB.x = D_i2_80195740[arg1];
+    srcB.y = D_i2_80195750[arg1];
+    srcB.z = D_i2_80195760[arg1];
+    Matrix_MultVec3fNoTranslate(gCalcMatrix, &srcA, &destA);
+    Matrix_MultVec3fNoTranslate(gCalcMatrix, &srcB, &destB);
+
+    Actor_Initialize(actor);
+    actor->obj.pos.x = player->pos.x + destA.x;
+    actor->obj.pos.y = player->pos.y + destA.y;
+    actor->obj.pos.z = player->unk_138 + destA.z;
+    actor->fwork[0] = destB.x;
+    actor->fwork[1] = destB.y;
+    actor->fwork[2] = destB.z;
+    actor->fwork[7] = Rand_ZeroOne() * 360.0f;
+    actor->fwork[8] = Rand_ZeroOne() * 360.0f;
+    actor->vel.x = player->vel.x;
+    actor->vel.y = player->vel.y;
+    actor->vel.z = player->vel.z;
+    actor->obj.status = 1;
+    actor->obj.id = OBJ_ACTOR_195;
+    Object_SetInfo(&actor->info, actor->obj.id);
+    if (arg1 == 3) {
+        actor->unk_0B6 = 1;
+        actor->unk_0B8 = 0x14;
+        actor->obj.rot.x = -player->unk_0E4 - 10.0f;
+        actor->obj.rot.y = (player->unk_0E8 + 180.0f) - 10.0f;
+        actor->fwork[9] = 10.0f;
+        return;
+    }
+    actor->obj.rot.z = D_i2_80195770[arg1];
+    actor->iwork[0xB] = 1;
+    actor->iwork[0xE] = D_i2_8019577C[arg1];
```

Comment:
```suggestion
    actor->iwork[11] = 1;
    actor->iwork[14] = D_i2_8019577C[arg1];
```
work indices also in decimal